source/includes/warn-console-stream.rst

```diff
@@ -0,0 +1,5 @@
+.. important::
+
+   Streaming to your console is memory intensive and intended 
+   only for testing purposes. Avoid streaming large datasets 
+   to your console.
```

Comment:
Suggestion: Reverse the order of the sentences. I think explaining the "why" factor first is a bit confusing if the "what" factor isn't presented first.

Reply:
Good call